Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -169,6 +169,18 @@ public enum SentrySDK {
         }
 
         let stowedExceptionCode = 0xC000027B
+        var context = CONTEXT()
```

Comment:
I think probably move this inside the else {} so that it doesn't affect the captureStowedExceptions call?